website/docs/basics/helpers-javascript.md

```diff
@@ -215,3 +215,22 @@ Converts the first letter of a string to uppercase.
 
   ucfirst('custom');
 ```
+
+## camelize
+
+Returns a camel-cased string.
```

Comment:
I think it's important to mention that this 'camelize' helper only works if you have a dash in the name. So if you put `some name` it won't turn it into `someName`. That's because it's meant to work with the libs where we define the name of the component as `component-name`.